tests/cypress/support/commands_organizations.js

```diff
@@ -31,9 +33,10 @@ Cypress.Commands.add('createOrganization', (organizationParams) => {
 Cypress.Commands.add('deleteOrganizations', (authResponse, otrganizationsToDelete) => {
     const authKey = authResponse.body.key;
     cy.request({
-        url: '/api/v1/organizations?page_size=all',
+        url: '/api/organizations?page_size=all',
         headers: {
             Authorization: `Token ${authKey}`,
+            Accept: acceptHeader,
```

Comment:
@Marishka17 , @bsekachev , should UI work with the latest version (=default) of REST API always? Thus we can skip acceptHeader for tests and UI.